[WPE] `TestUIClient` `/webkit/WebKitWebView/geolocation-permission-requests` API test times out

obyknovenius opened the pull request:
#### 3628406acf14f1e985fcebf7a5e31d183280400e
<pre>
[WPE] `TestUIClient` `/webkit/WebKitWebView/geolocation-permission-requests` API test times out
<a href="https://bugs.webkit.org/show_bug.cgi?id=261199">https://bugs.webkit.org/show_bug.cgi?id=261199</a>

Unreviewed test gardening.

For this test to finish one of the
`navigator.geolocation.getCurrentPosition()` callbacks need to be
called, either the success or the error.

When we don&apos;t have access to `org.freedesktop.GeoClue2` DBus service,
the error callback will be called quite soon.

When we do have access, the time needed to get the current position can
exceed the default 5 seconds we give for the test to finish.

This test times out only on WPE Release bot. This is the only Linux bot
running on bare hardware, not in a container. It seems like the
container environment doesn&apos;t provide access to the GeoClue DBus server.
That explains why we don&apos;t get the timeout on other Linux bots.

Tools/TestWebKitAPI/glib/TestExpectations.json:

Canonical link: <a href="https://commits.webkit.org/267736@main">https://commits.webkit.org/267736@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/931b98a05c1e04237c19d3f914a630ee30822bdc

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/17540 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/17867 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/18389 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/19345 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/16409 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/17737 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/21148 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/18011 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/18524 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/17748 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/18056 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/15228 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/20169 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/15290 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/15964 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/22581 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/16300 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/16133 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/20431 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/16712 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/14154 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/15834 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/4175 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/20201 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/16552 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/94136f945aa0134006650cc9d8d70d5fb30adf36. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/17478

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/e24530404864f9eca07bc8a0b343934482e7418e. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/17478

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/931b98a05c1e04237c19d3f914a630ee30822bdc. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/17478

obyknovenius committed:
[WPE] `TestUIClient` `/webkit/WebKitWebView/geolocation-permission-requests` API test times out
https://bugs.webkit.org/show_bug.cgi?id=261199

Unreviewed test gardening.

For this test to finish one of the
`navigator.geolocation.getCurrentPosition()` callbacks need to be
called, either the success or the error.

When we don't have access to `org.freedesktop.GeoClue2` DBus service,
the error callback will be called quite soon.

When we do have access, the time needed to get the current position can
exceed the default 5 seconds we give for the test to finish.

This test times out only on WPE Release bot. This is the only Linux bot
running on bare hardware, not in a container. It seems like the
container environment doesn't provide access to the GeoClue DBus server.
That explains why we don't get the timeout on other Linux bots.

Tools/TestWebKitAPI/glib/TestExpectations.json:

Canonical link: https://commits.webkit.org/267736@main

webkit-commit-queue commented:
Test gardening commit 267736@main (3628406acf14): <https://commits.webkit.org/267736@main>

Reviewed commits have been landed. Closing PR #17478 and removing active labels.
